lib/WUI/netif_settings.h

```diff
@@ -24,6 +24,16 @@ typedef enum {
     ETH_NETIF_UP,   // ETH interface is UP
 } ETH_STATUS_t;
 
```

Comment:
#pragma once
convert macro functions to typesafe procedures:
```
turn_lan_on(uint8_t *flag){
  flag.on = 1;
}
```

Reply:
could be part of another round of refactoring, there are more